### DISCUSSION TITLE
Prefix IP Listing

### DISCUSSION THREAD

**Discussion by bmp521** [netbox-community/netbox#10444] [Q&A] [answered]
I have recently noticed that when I select a prefix, the available IP address rows are all listed at the top of the table when before they were throughout the table and showed the gaps between assigned IP addresses. I can't see anyway that I have sorted the table to group them all at the top. However, if I do sort the list by IP address, I lose all the rows that show the available IP addresses. I'm not sure if its just an issue with 3.3.2 since I upgraded from 3.2.7 about a week ago. Has anyone else run into this issue or have any suggestions?

![IP sort issue](https://user-images.githubusercontent.com/40783313/191820294-04f4505c-38af-4aed-8d49-41025da8e48a.PNG)

**Comment by kkthxbye-code** [comment, marked as answer]
What happens if you press the red x?

**Reply by bmp521** [reply]
Thank you! That fixed the issue. I don't recall how I even set that option/flag.
